src/datapilot/clients/altimate/constants.py

```diff
@@ -0,0 +1,9 @@
+"""Constants for the Altimate API client."""
+
+# Supported dbt artifact file types for onboarding
+SUPPORTED_ARTIFACT_TYPES = {
+    "manifest",
+    "catalog",
+    "run_results",
+    "sources",
```

Comment:
also add semantic manifest, we can already collect it, we should not validate it.